qiita_core/support_files/config_test.cfg

```diff
@@ -42,6 +42,9 @@ VALID_UPLOAD_EXTENSION = fastq,fastq.gz,txt,tsv,sff,fna,qual
 # Portal the site is working under
```

Comment:
Since these have to do with portals, shouldn't these be under the portal section of the config file?